zipline-loader/src/jvmTest/kotlin/app/cash/zipline/loader/ZiplineDownloaderTest.kt

```diff
@@ -91,4 +93,34 @@ class ZiplineDownloaderTest {
       testFixturesJvm.bravoByteString,
       fileSystem.read(downloadDir / testFixturesJvm.bravoSha256Hex) { readByteString() })
   }
+
+  @Test
+  fun integrationTest(): Unit = runBlocking(dispatcher) {
+    assertFalse(fileSystem.exists(downloadDir / PREBUILT_MANIFEST_FILE_NAME))
+    assertFalse(fileSystem.exists(downloadDir / testFixturesJvm.alphaSha256Hex))
+    assertFalse(fileSystem.exists(downloadDir / testFixturesJvm.bravoSha256Hex))
+
+    val baseUrl = "http://10.0.2.2:8080/".toHttpUrl()
+    downloader = ZiplineDownloader(
+      dispatcher = dispatcher,
+      httpClient = OkHttpZiplineHttpClient(baseUrl, OkHttpClient()),
+      downloadDir = downloadDir,
+      downloadFileSystem = fileSystem,
+    )
+    downloader.download("https://treelot.sqprod.co/latest/treehouse-activity/manifest.zipline.json")
```

Comment:
Needs to be served via MockWebServer and not hit the real internet

Reply:
Thanks, I'll set up a test that does that